README.md

```diff
@@ -78,6 +78,17 @@ alias zq="zoxide query"
 alias zr="zoxide remove"
 ```
 
+#### fish
+Using [fisher](https://github.com/jorgebucaran/fisher):
+```sh
+fisher add ajeetdsouza/zoxide
+```
+
+Using [omf](https://github.com/oh-my-fish/oh-my-fish):
```

Comment:
Could you change `omf` to `oh-my-fish` for clarity?

Reply:
Done!